===================================================================
RCS file: /srv/anoncvs.freedesktop.org/cvs/xorg/lib/Xfixes/src/Xfixes.c,v
rcsdiff: /srv/anoncvs.freedesktop.org/cvs/xorg/lib/Xfixes/src/Xfixes.c,v: warning: Unknown phrases like `commitid ...;' are present.
retrieving revision 1.3
retrieving revision 1.4
diff -u -r1.3 -r1.4
--- src/Xfixes.c	2005/05/17 17:02:24	1.3
+++ src/Xfixes.c	2006/04/03 00:58:13	1.4
@@ -86,6 +86,7 @@
 	{
 	    UnlockDisplay (dpy);
 	    SyncHandle ();
+	    Xfree(info);
 	    return 0;
 	}
 	info->major_version = rep.majorVersion;
